lib/survey_gizmo/resource.rb

```diff
@@ -137,7 +137,7 @@ def logger
     # Returns itself if successfully saved, but with attributes (like id) added by SurveyGizmo
     def save
       method, path = id ? [:post, :update] : [:put, :create]
-      self.attributes = Connection.send(method, create_route(path), attributes_without_blanks).body['data']
+      self.attributes = Connection.send(method, create_route(path), attributes_without_blanks).body['data'].to_h
```

Comment:
Good point, I prefer too, it wouldn't handle the empty array case from #70 though ;)